lib/src/Uncertainty/Algorithm/Sensitivity/HSICEstimatorImplementation.cxx

```diff
@@ -282,29 +282,20 @@ Graph HSICEstimatorImplementation::drawValues(const Point &values, const String
   Text text(data, names, "right");
   text.setColor("black");
   graph.add(text);
-  //graph.setXMargin(0.9);
-  //graph.setYMargin(0.9);
 
   // Set bounding box
+  const Scalar step = maxInd - minInd;
+
   Point lowerBound(2);
   lowerBound[0] = 0.9;
-  if (minInd < 0)
-    lowerBound[1] = 1.1 * minInd;
-  else
-    lowerBound[1] = 0.9 * minInd;
+  lowerBound[1] = minInd - 0.05*step;
 
   Point upperBound(2);
-  upperBound[0] = values.getDimension() + 0.1 ;
-
-  if (maxInd > 0)
-    upperBound[1] = 1.1 * maxInd;
-  else
-    upperBound[1] = 0.9 * maxInd;
+  upperBound[0] = values.getDimension() + 0.2 ;
+  upperBound[1] = maxInd + 0.05*step;
 
   graph.setBoundingBox(Interval(lowerBound, upperBound));
```

Comment:
the default is already to add a 5% margin, doesnt removing the setBoundingBox produce the same graph ?

Reply:
No, I don't think so. Otherwise, @mbaudin47 would have been happy with the default graph, right?